.coveragerc

```diff
@@ -99,8 +99,6 @@ omit =
     homeassistant/components/brunt/cover.py
     homeassistant/components/bt_home_hub_5/device_tracker.py
     homeassistant/components/bt_smarthub/device_tracker.py
-    homeassistant/components/buienradar/sensor.py
```

Comment:
Are we covering enough to remove these? We need more at least 95%.

Reply:
Definitely not at 95%. Have re-added this back in and added `util`. Above that threshold for camera; much lower for `util`, `sensor`, and `weather`.

Think this is why CI failed?

Unfortunately I cannot run the full test suite on my workstation (linux, many dependency issues) so I can not run tox 😔.